README.md

```diff
@@ -1,5 +1,7 @@
 # Git Merge Information
 
+Discord: [![Discord](https://dcbadge.limes.pink/api/server/v2FqAaWX)](https://discord.com/channels/1042895022950994071/1265007518942367755)
```

Comment:
The channel link is not a server invitation. It'll only work for people already on the Git Community server.

Reply:
Plus: it seems that the actual chat will happen on GitButler's Discord.